test/integration-serial/serial_deploy_test.rb

```diff
@@ -209,15 +209,11 @@ def test_stage_related_metrics_include_custom_tags_from_namespace
 
   def test_all_expected_statsd_metrics_emitted_with_essential_tags
     metrics = capture_statsd_calls do
-      result = deploy_fixtures('hello-cloud', subset: ['configmap-data.yml'], wait: false)
+      result = deploy_fixtures('hello-cloud', subset: ['configmap-data.yml'], wait: false, sha: 'test-sha')
       assert_deploy_success(result)
     end
 
     assert_equal 1, metrics.count { |m| m.type == :_e }, "Expected to find one event metric"
-    assert_predicate metrics, :all? do |metric|
```

Comment:
This isn't doing what it appears to be doing. If it were, it would have caught the missing tags. Better to check on the explicit list of metrics below anyway.